roles/bootstrap/tasks/directories.yml

```diff
@@ -0,0 +1,12 @@
+---
+
+- name: Create required system directories
+  file:
+    path: "{{ item }}"
+    state: directory
+  become: true
+  with_items:
+    - /data
+    - /srv
+    - /var/log
```

Comment:
bike-shedding.... is /var/log missing from the base ami ?

Reply:
No it's not, will remove